app/models/block/main_navigation.rb

```diff
@@ -0,0 +1,17 @@
+module Block
+  class MainNavigation < Block::Base
+    attr_reader :title, :title_link, :links
+
+    def initialize(block_hash)
+      super(block_hash)
+
+      @title = data.fetch("title")
```

Comment:
To be honest, I'm not sure if this model is really needed as the you can just use `block.data["title"]` etc in the views. However being able to call `block.title` is tidier.